### ISSUE TITLE
[v0.7.0] keyword 'self', 'parent', 'static' call static function not working inside Class

### ISSUE THREAD

**Issue by kocoten1992** [bobthecow/psysh#288]
```
class A {
    private function foo() {
        echo __CLASS__;
    }
    public function test() {
        static::foo();
    }
}
```

Error: PHP Fatal error:  Call to undefined method static::foo() in eval()'d code on line 6

It execute ASAP, regardless inside class, test with v0.6.1 and v0.7.0

**Comment by bobthecow** [comment]
Thank you for reporting. This has been fixed in the develop branch and will go out in the next point release.

**Comment by bobthecow** [comment]
This fix is included in v0.7.1.
